app/forms/mappings/Formatters.scala

```diff
@@ -24,13 +24,17 @@ import scala.util.control.Exception.nonFatalCatch
 
 trait Formatters {
 
-  private[mappings] def stringFormatter(errorKey: String): Formatter[String] = new Formatter[String] {
+  def replaceSpaces (input: String): String = input.trim.replaceAll(" +", " ")
+
+  def trimString (input: String): String = input.trim.replaceAll(" +", " ")
+
+  private[mappings] def stringFormatter(errorKey: String, fun: String=>String = trimString): Formatter[String] = new Formatter[String] {
```

Comment:
could you please give `fun` a more descriptive name (even just `function`, just to avoid confusion for people coming from languages that use 'fun' in place of 'def'?)

Reply:
also please add a space before & after the rocket